android/src/main/java/ru/innim/flutter_login_facebook/Results.java

```diff
@@ -22,6 +22,13 @@
         }};
     }
 
+    public static HashMap<String, Object> loginSuccess(final AccessToken accessToken) {
```

Comment:
`loginSuccess(final LoginResult result)` should call this method since their both returns same object (`FacebookLoginResult`)